deps/build.jl

```diff
@@ -125,11 +138,55 @@ copy_config("ijuliafavicon.ico",
 # custom.js can contain custom js login that will be loaded
 # with the notebook to add info and/or monkey-patch some javascript
 # -- e.g. we use it to add .ipynb metadata that this is a Julia notebook
-copy_config("custom.js", joinpath(juliaprof, "static", "custom"))
+if ipyvers <= v"3.0-"
```

Comment:
Why is there an `if` statement here if both branches of the `if` do the same thing?